WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -574,6 +590,20 @@ private extension SettingsViewController {
         NotificationCenter.default.post(name: NSNotification.Name(rawValue: "wormholy_fire"), object: nil)
     }
 
+    func whatsNewWasPressed() {
+        ServiceLocator.stores.dispatch(AnnouncementsAction.loadSavedAnnouncement(onCompletion: { [weak self] result in
+            guard let self = self else { return }
+            guard let (announcement, _) = try? result.get() else {
+                return DDLogInfo("📣 There are no announcements to show!")
+            }
+
+            let viewController = WhatsNewFactory.whatsNew(announcement) { [weak self] in
+                self?.dismiss(animated: true)
+            }
+            self.present(viewController, animated: true, completion: nil)
+        }))
```

Comment:
I think this could show an old announcement from a past app version right?
I'm sure some app versions won't have announcements because they don't have any significant user facing change.

Reply:
Yea, I thought for a second that we were going to always display the latest announcement regardless of the app version. But it may confuse users if for example we remove a feature and tell users that this is "new on WooCommerce". 
I've added a check for the app version! Thanks for the head's up!

Done! ✅